templates.go

```diff
@@ -64,7 +64,7 @@ func NewTemplate(co *Company, in *CV) *Page {
 			func() *Element {
 				ul := Ul()
 				for _, edu := range in.Educations {
-					ul.With(Li(edu.Subject), periodSpan(edu.Period))
+					ul.With(Li(edu.Grade, " in ", edu.Subject), periodSpan(edu.Period))
```

Comment:
According to https://standout-cv.com/how-to-write-your-degree-on-your-cv  the binding word is "of".

Reply:
Sorry went a head of my self here. Bachelor of science but should it not be in computer science? or do you mean it should be Bachelor of science of Computer Science?